app/services/csv_exporter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+module CsvExporter
+  def self.to_csv(resources, columns)
+    Enumerator.new do |csv|
+      csv << csv_header(columns)
+
+      resources.find_each do |obj|
+        csv << csv_row(obj, columns)
```

Comment:
I think `obj` here has to be flattened. Otherwise relations (e.g: column_names for Product has `sync_plan`, but `sync_plan` by itself will just return an object) will not show up.

If it's not flattened, we will have to write a `to_s` for everything that doesn't display well as a string